### DISCUSSION TITLE
How do I populate a `Formula` upon entity creation?

### DISCUSSION THREAD

**Discussion by ahmedelq** [mikro-orm/mikro-orm#5508] [Q&A]
So in my order entity I have a formula which finds the total (quantity * unit price) as such:
```typescript
 @Formula(
    (alias) =>
      `(select sum(oi.count * oi.unit_price) from order_item oi where oi.order_id = ${alias}.id)`,
    { lazy: false }
  )
  total?: number;
```
However, when I create using using the below, the response I get doesn't contain `total`. 

```typescript
const orderObj = await em.create(OrderEntity, {...});
await em.persistAndFlush(orderObj);
wrap(orderObj).populate(["total"])
```

**Comment by B4nan** [comment]
Lazy: false is the default, you dont need to populate that at all. Hard to help without seeing a complete repro.
